db/re/skill_db.conf

```diff
@@ -4520,6 +4520,7 @@ skill_db: (
 	Element: "Ele_Wind"
 	DamageType: {
 		SplashArea: true
+		SplitDamage: true
```

Comment:
need to add at the top of skill_db.conf what SplitDamage is, boolean etc.

Reply:
it's there already :) https://github.com/HerculesWS/Hercules/blob/a3c811a82712fa80c1a8481a54561fb38390638a/db/re/skill_db.conf#L91